docs/performance.md

```diff
@@ -7,11 +7,30 @@
 
 The top question about the performance of CORE is often *how many nodes can it handle?* The answer depends on several factors:
 
-* Hardware - the number and speed of processors in the computer, the available processor cache, RAM memory, and front-side bus speed may greatly affect overall performance.
-* Operating system version - distribution of Linux and the specific kernel versions used will affect overall performance.
-* Active processes - all nodes share the same CPU resources, so if one or more nodes is performing a CPU-intensive task, overall performance will suffer.
-* Network traffic - the more packets that are sent around the virtual network increases the amount of CPU usage.
-* GUI usage - widgets that run periodically, mobility scenarios, and other GUI interactions generally consume CPU cycles that may be needed for emulation.
+<table>
```

Comment:
Before you go too far with updating them all to HTML, I rather we kept it in Markdown. It does appear that there is no native support for tables without headers. I would just add actual header text for now so we can keep it in Markdown format and apply that to all the tables below.